tasks/README.rst

```diff
@@ -56,105 +55,92 @@ C) Get creative! Right after the inline comments, add a second door using the el
+now print the value of `valid` options that are passed. For example if the valid options for the user to type is `1` or `2`,
+when the user types something other than 1 or 2, the `wrong_input` function should be called like that:
 
 .. code-block:: python
 
-    # Exercise A
+    wrong_input("1, 2") # this could print: "Sorry but the only valid options are: 1,2. Try again please"
 
-    # Exercise C
 
-    print()
-    # Your code from Exercise A on Task 2 should be here
+4. Loops and Lists
+==================
 
-    # Your code from Exercise B on Task 2 should be here
+Continue in your file from the previous exercise or use the solution from the previous exercise
+as a base for this exercise.
 
-    print("Do you go through door 1 or door 2?")
+A) At the beginning of the file, create a *list* variable named `friends` with names of the user's friends.
 
-    door = input("> ")
+    # Remember defining a list variable looks like that:
+    my_var = ['zero_element', 'first_element']
 
-    if door == "1":
-        print("There is a nice vampire asking you if you enjoy life.")
-        print("What do you do?")
-        print("1. Smile and nod")
-        print("2. Scream and run")
+B) When a user enters some room, print a message saying that her 2nd friend in the list is in the room (print the name
+   of the friend from the variable `friends`).
 
-        vampire = input("> ")
+    # Remember: Access list elements like this eg. friends[0], friends[1]
 
-        if vampire == "1":
-            print(f"Congratulations {name}, you found a new friend!")
-        elif vampire == "2":
-            print(f"Sorry {name}, the vampire is faster. You become a dinner.")
-        else:
-            # Exercise B
+C) Make this name to be chosen randomly. At the first line of the file, import Python library called `random`.
+This library has functions to help with random numbers and use `random.randint(a, b)` function to give you a
+random number between a and b. eg. `random.randint(1, 4)` will return a random number between 1-4, so one of
+1, 2, 3, 4. Use this function to choose a random name from the names list.
 
-    # Your code from Task 1 should be here
+    # Remember: import statement looks like this:
+    import that_awesome_library_name
 
-    else:
-        wrong_input()
 
-    # Your code from Exercise C on Task 2 should be here
+5. Loops, Range, Function
+=========================
 
+Continue in your file from the previous exercise or use the solution from the previous exercise
+as a base for this exercise.
 
-4. Loops and Lists
-==================
+A) Now think a bit, how would you write a `dead` function using an argument called `death_message`? This
+ function could be called in every point that they user dies eg. when the user becomes a dinner for the
+  vampire. To figure out where this function should be added, check the structure of a typical python file:
+
```

Comment:
the lines should start at the same point, otherwise it becomes a citation.

Reply:
what do you mean exactly here?

Reply:
see that there's space at the beginning of lines 130 and 131? those spaces are been interpreted as a formatting for citation.

Reply:
fixed